src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -31,6 +31,15 @@ abstract class PipelineTargets(config: Config) {
       partitionBy = Seq("organization_id")
     )
 
+    lazy private[overwatch] val databricksSqlTarget: PipelineTable = PipelineTable(
+      name = "databricks_sql_snapshot_bronze",
+      _keys = Array("id", "Overwatch_RunID"),
+      config,
+      incrementalColumns = Array("Pipeline_SnapTS"),
+      statsColumns = "created_time, creator_user_name, id, Pipeline_SnapTS, Overwatch_RunID".split(", "),
+      partitionBy = Seq("organization_id")
+    )
```

Comment:
as you more clearly define which table goes where, just be diligent on making sure the various columns are accurate. It becomes very important later and it wills ave you a lot of headaches. From what I see in the structure, several of those stats columns do not exist